Lib/test/test_tarfile.py

```diff
@@ -738,31 +738,6 @@ def test_extract_directory(self):
         finally:
             os_helper.rmtree(DIR)
 
-    def test_deprecation_if_no_filter_passed_to_extractall(self):
```

Comment:
A new test case to check that the default is `data_filter` would also be nice :)

Reply:
My buddy @sodle just added one:
https://github.com/python/cpython/pull/122002/commits/15216b9aa19aeb3d316c416b64139637bb54dd75